### ISSUE TITLE
hub.docker images don't yet have the minitest gem

### ISSUE THREAD

**Issue by siliconmeadow** [Starefossen/docker-github-pages#13]
Hi @Starefossen - I see that you created a number of tags that have the different releases of the gh-pages gem, but the error reported in issue #11 still exists with your latest image as it appears the minitest gem is still not being added.

I could be wrong, but that's what appears to be the problem to me.

**Comment by Starefossen** [comment]
I have triggered a new build on Docker Hub, lets see if this fixes the problem.

**Comment by siliconmeadow** [comment]
It seems to be something specific to my site, I guess. I just created a new local site instead, swapped the `jekyll` gem for the `github-pages` gem and it works.

Prior to your updates today, I created my own image at `siliconmeadow/docker-github-pages` (which adds that `minitest` gem in the `RUN` command) and it's working ok.

I'm still relatively new to docker, so I'm nearly exhausted all my ideas for troubleshooting at this stage.
